xml/System/Object.xml

```diff
@@ -420,15 +420,17 @@
  If <xref:System.Object.Finalize%2A> or an override of <xref:System.Object.Finalize%2A> throws an exception, and the runtime is not hosted by an application that overrides the default policy, the runtime terminates the process and no active `try`/`finally` blocks or finalizers are executed. This behavior ensures process integrity if the finalizer cannot free or destroy resources.  
   
 <a name="Notes"></a>   
-## Notes for implementers  
- You should override <xref:System.Object.Finalize%2A> for a class that uses unmanaged resources such as file handles or database connections that must be released when the managed object that uses them is discarded during garbage collection.  
+## Overriding the Finalize method 
+ You should override <xref:System.Object.Finalize%2A> for a class that uses unmanaged resources such as file handles or database connections that must be released when the managed object that uses them is discarded during garbage collection. You should not implement a <xref:System.Object.Finalize%2A> method for managed objects, bevause the garbage collector releases managed resources automatically.  
```

Comment:
Comma before the "such as" setting of the list of examples ...

> ... uses unmanaged resources, such as file handles or database ...

"should not" :point_right: "shouldn't"
"bevause" :point_right: "because" (spelling)

... probably also remove the comma before "because" ... the reasoning isn't ambiguous for that sentence.